tools/lsp/common.rs

```diff
@@ -154,10 +156,6 @@ pub enum LspToPreviewMessage {
         url: Option<Url>,
         offset: u32,
     },
-    KnownComponents {
```

Comment:
I'm not sure it's such a good idea to compute the known component from the preview side: you only get what is actually used by the preview. But I think it is better to get the full lost of component for the whole project.